pkg/reconciler/v1alpha1/autoscaling/kpa/kpa.go

```diff
@@ -317,11 +324,15 @@ func (c *Reconciler) reconcileDecider(ctx context.Context, pa *pav1alpha1.PodAut
 func (c *Reconciler) reconcileSKS(ctx context.Context, pa *pav1alpha1.PodAutoscaler) (*nv1alpha1.ServerlessService, error) {
 	logger := logging.FromContext(ctx)
 
+	mode := nv1alpha1.SKSOperationModeServe
+	if pa.Status.IsInactive() {
```

Comment:
So this keys off of the activity state of the PA via status, but the inactivity is reflected in the PA's status *after* we call reconcileSKS, so IIUC it will always take two reconciles to put it into proxy mode.  Ideally it wouldn't.

Reply:
As discussed in person. I have some ideas how to make it reconcile twice slightly less often (or perhaps never), but let's get his working and then deal with optimizations later.
For example, we can try to reconcile decider first and then feed it's 0 into the SKS, but this also requires to reconcile scaler first, and then stuff cascades.